doc/configuration-v3_2_experimental.md

```diff
@@ -131,6 +131,10 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_options_** (list of strings): any additional options to be passed to the cryptsetup utility.
     * **_wipeVolume_** (boolean): whether or not to wipe the device before volume creation, see [the documentation on filesystems](operator-notes.md#filesystem-reuse-semantics) for more information.
     * **_clevis_** (object): describes the clevis configuration for the luks device.
+      * **_custom_** (object): override the clevis configuration. The `pin` & `config` will be passed directly to `clevis luks bind`. If specified all other clevis options must be excluded.
```

Comment:
Since we're not constrained to list fields alphabetically, let's put this after the other fields.

`overrides`

`If specified, all other clevis options must be omitted.`